generators/app/templates/src/StarterKit/Api/Controllers/ValuesController.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
```

Comment:
zelfde opmerking als bij DbExamplesController. Dit bestand zou beter geen onderdeel vormen van de starterkit, want dat komt dit bij elke generatie terug in het project te zitten.